### ISSUE TITLE
IE11 and Edge broken with latest release

### ISSUE THREAD

**Issue by stramel** [Polymer/polymer-cli#702]
### Description

Please don't hate me for this. I am still digging into it but can say for certain that v0.17.0 serving my build directory works as expected in IE11 and Edge but with v0.18.1 it doesn't.

Errors that I am getting:
`@font-face failed WOFF integrity check.`
`No fonts available for @font-face rule`
`Unable to decode image at URL: 'http://localhost:8081/images/logos/citrix-main.png' with MIME: 'image/png'.`
`Unable to decode image at URL: 'http://localhost:8081/images/backgrounds/login.jpg' with MIME: 'image/jpeg'.`

I will work on reproduction steps for this. 

### Versions & Environment

- Polymer CLI: v0.18.1
- node: v6.9.1
- Operating System: Windows 10

**Comment by justinfagnani** [comment]
@stramel I assume that you're serving the built directories with `polymer serve` since you suspect it. Can you serve with `polymer serve --compile-never` (after you've build with `polymer build`)?

**Comment by stramel** [comment]
@justinfagnani Yes, I am serving the built directory. (`polymer serve build/default`). Let me check what you just suggested and post back with the results.

**Comment by stramel** [comment]
@justinfagnani `polymer serve --compile-never` results in the following error:

```
 $ polymer serve --compile-never
error:   cli runtime exception: UNKNOWN_OPTION: Unknown option: --compile-never
```

**Comment by justinfagnani** [comment]
Sorry, that should be `--compile=never`

**Comment by stramel** [comment]
Updating this thread with what was discussed on Slack for the record.

I tried using an alternative serving tool (`http-server`) and that worked.

Using @justinfagnani's suggested command `polymer serve --compile=never build/default/ -o` worked as well.

**Comment by davidmaxwaterman** [comment]
From memory, I would say the option should be `--compile never`, but I can't find that in the help in v1.0.2 :/ The other options, though, all fit that pattern. IMO, it should be `--compile=never` or `-c never`, but polymer-cli doesn't seem to follow the standard (which is a pretty loose standard, admittedly). I'm not sure what is happening, since it should be in the help, right?

**Comment by srijken** [comment]
We're running into a similar problem with PNG files not working. Appears to be related to the User-Agent header in the GET request. When this is similar to IE11 we get about double the data in the response compared to Chrome User-Agent. 

This was tested using Fiddler, so no other variables were changed that might have effect.
